src/volume/ndarray.rs

```diff
@@ -103,6 +103,6 @@ where
     {
         // TODO optimize this implementation (we don't need the whole volume)
         let volume = self.volume.into_ndarray()?;
-        Ok(volume.into_subview(Axis(self.axis as Ix), self.index as usize))
+        Ok(volume.index_axis_move(Axis(self.axis as Ix), self.index as usize))
```

Comment:
Is this change necessary for the updated crates to work? Since this method was only introduced in recent versions of `ndarray`, I was hoping to choose the right moment to make this transition.

Reply:
No everything was fine, just a warning during the compilation.
```
warning: use of deprecated item 'ndarray::impl_methods::<impl ndarray::ArrayBase<S, D>>::into_subview': renamed to `index_axis_move`
```

I can revert it if you prefer.

Reply:
Yes please, if you don't mind. I am aware of this quirk, but changing the method name here requires a different range of compatible `ndarray` versions, and I'd like to record this change separately.